zuul.d/jobs.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- job:
+    name: base-crc-openstack
+    parent: base-crc
+    timeout: 10800
+    nodeset: centos-8-crc-single-node-centos-9-external-compute
```

Comment:
note, we now have a CRC node running on cs9 - it's working just fine, used in ci-framework job. Maybe something to consider?